check/tests/support/mod.rs

```diff
@@ -1,66 +1,71 @@
-extern crate env_logger;
+#![allow(private_in_public, dead_code)]
```

Comment:
I was getting errors with the reexports of the crates:

Eg.

```
warning: extern crate `gluon_base` is private, and cannot be reexported (error E0364), consider declaring with `pub`, #[warn(private_in_public)] on by default
 --> /Users/brendan/dev/rust/gluon/check/tests/support/mod.rs:7:9
  |
7 | pub use self::gluon_base as base;
  |         ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
  = note: for more information, see the explanation for E0446 (`--explain E0446`)
```

I couldn't put a `pub` on the `extern crate` though. I didn't see the deprecation notice in my haste though. I'll move the crate imports to the top instead.